nextjs-13/src/app/api/v1/me/route.test.ts

```diff
@@ -4,58 +4,37 @@
 
 import { User } from '@prisma/client';
 import { StatusCodes } from 'http-status-codes';
-import * as nextAuthJwtModule from 'next-auth/jwt';
 
 import { dbClientMock } from '@test/database';
 import { userFactory } from '@test/factories/user.factory';
+import baseHandler from 'lib/handler/base.handler';
 
 import { GET } from './route';
+jest.mock('lib/handler/base.handler');
```

Comment:
I think you should I add the line-break between the `import` and `jest.mock`

Reply:
Fixed [here](https://github.com/nimblehq/js-framework-benchmark/pull/96/commits/7e83686ee97e415876c573b274caaf60c3984dd6#diff-2ee6b5c4cbf359c8cda60ecacf45024db9a86bbe0eaa6ad5d51dfb7ee9780aa4R13)